auto_tune_vllm/core/study_controller.py

```
@@ -0,0 +1,586 @@
+                    logger.info("Database already exists")
+            except Exception as e:
+                logger.error(f"Failed to create database: {e}")
+                raise RuntimeError(f"Database creation failed: {e}")
+        
+        # Verify database connection before proceeding (only if using PostgreSQL)
+        if config.database_url and not verify_database_connection(config.database_url):
+            if create_db:
+                raise RuntimeError(
+                    f"Cannot connect to database after creation attempt. "
+                    f"Please check your database URL: {config.database_url}"
+                )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)